verify/cancel.php

```diff
@@ -1,9 +1,12 @@
-<?php 
+<?php
```

Comment:
Same thing with this, not sure anyone would notice this otherwise, but some of the classes you have the new line empty after `<?php` but others there isn't an empty line.

Reply:
Fixed. For consistency there should be a line after the opening `<?php` statement.